src/ElementalChat.vue

```diff
@@ -215,21 +215,29 @@ export default {
     handleToDisplay () {
       return this.isHoloAnonymous ? 'anonymous' : this.agentHandle
     },
+    shouldAutoSignIn () {
+      return isHoloHosted() && !isAnonymousEnabled() && this.holoStatus === 'ready' && this.isHoloAnonymous
```

Comment:
Can we name this a something like `isDefaultAnonymousEnabled`?  I know it's longer and not ideal, but it would be nice to clearly distinguish between the fact of being in an anonymous state (ie: `isHoloAnonymous` === true) and the default anonymous variable being true or false.

Reply:
> the default anonymous variable being true or false.

My intention is that if anonymous is disabled, and you click Logout, it takes you back to the log in screen. Meaning that it's not just the default state but rather it determines whether or not a user is allowed to interact with the site anonymously. I think enabled is a pretty good way of saying that

Reply:
I'm fine with that then, but would just request an inline note just above this line or so, stating something similar to you said here.  The goal is to help clarify the what it is intended when `isAnonymousEnabled()` is false, but `this.isHoloAnonymous` is true.